pkg/helm/client.go

```diff
@@ -128,8 +128,10 @@ func (h CliClient) UpdateRepo(ctx context.Context) error {
 	return nil
 }
 
-// Install installs helm chart with given release name
-func (h CliClient) Install(ctx context.Context, chart, version, release, namespace string, values map[string]string, wait bool) error {
+// Install installs helm chart with given release name.
+// The output string is to check the generated manifests of a release without installing the chart when ‘dryRun’ flags combined.
+// For normal installation, this output string is blank.
+func (h CliClient) Install(ctx context.Context, chart, version, release, namespace string, values map[string]string, wait, dryRun bool) (out string, err error) {
```

Comment:
is it intentional to have named return values in this function? I think if we don't have named return values here, we can return values when we use return statement and we will also not have to add `nolint.

Reply:
Yes, I did it intentionally. As I was following your suggested code change:
```	
if !dryRun {
		out, err := RunCmdWithTimeout(ctx, h.helmBin, cmd)
		if err != nil {
			log.Error().Print("Error installing helm chart", field.M{"output": out})
			return "", err
		}
		log.Debug().Print("Result", field.M{"output": out})
		return
	} 
```
I was getting the compilation error `not enough arguments to return`. Hence I had to introduced naked return.
Do let me know if you want to suggest something.

Reply:
in the above code can we not do 

```
return "", nil
```
because we were returning `nil` in the earlier version of the code?

Reply:
Ok